plone/app/vocabularies/images.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from plone.app.imaging.utils import getAllowedSizes
+from zope.schema.interfaces import IVocabularyFactory
+from zope.schema.vocabulary import SimpleTerm
+from zope.schema.vocabulary import SimpleVocabulary
+from zope.interface import provider
+
+
+@provider(IVocabularyFactory)
+def ScalesVocabulary(context):
+    """Obtains available scales from plone.app.imaging
+    """
+    terms = []
+    for scale, (width, height) in getAllowedSizes().iteritems():
+        terms.append(SimpleTerm(scale, scale, "{:s} {:d}x{:d}".format (
```

Comment:
- please use always numbers for the positions, like {0:s} {1:s}
- here some dynamic i18n message ids would be very useful, like ``_('imagescale_{0:s}.format(scale)', default='{0:s} ${width}x${height}'.format(scale), mapping={'width': str(width), 'height': str(height)})``` (untested)